packages/components/src/InputTime/InputTime.tsx

```diff
@@ -38,50 +45,16 @@ export function InputTime({
 
   function handleBlur() {
     params.onBlur?.();
+    console.log("handleBlur");
 
     // Time inputs doesn't clear the typed value when it's invalid. This should
     // force it to reset the input value when the typed value is invalid.
     if (ref.current) {
```

Comment:
another **existing issue** with this component is that it accepts an `inputRef` prop that gets applied to `FormField` in a non obvious way via `{...fieldProps}` where that spread is done after `inputRef={ref}` with a ref of our own that we're attempting to use

so, if you provide `inputRef` it will override the other `ref` leaving it `null` forever, and preventing this code block from ever executing

as I mention below, I'm not too clear on the implications of that since I don't know when it's supposed to trigger even when we have the ref.